observability/otel/otel-collector/otel-collector-processors.adoc

```diff
@@ -424,24 +424,26 @@ include::snippets/technology-preview.adoc[]
 [source,yaml]
 ----
 # ...
+mode: sidecar # <1>
 config: |
   processors:
     cumulativetodelta:
-      include: # <1>
-        match_type: strict # <2>
-        metrics: # <3>
+      include: # <2>
+        match_type: strict # <3>
+        metrics: # <4>
         - <metric_1_name>
         - <metric_2_name>
-      exclude: # <4>
+      exclude: # <5>
         match_type: regexp
         metrics:
         - "<regular_expression_for_metric_names>"
 # ...
 ----
-<1> Optional: Configures which metrics to include. When omitted, all metrics, except for those listed in the `exclude` field, are converted to delta metrics.
-<2> Defines a value provided in the `metrics` field as a `strict` exact match or `regexp` regular expression.
-<3> Lists the metric names, which are exact matches or matches for regular expressions, of the metrics to be converted to delta metrics. If a metric matches both the `include` and `exclude` filters, the `exclude` filter takes precedence.
-<4> Optional: Configures which metrics to exclude. When omitted, no metrics are excluded from conversion to delta metrics.
+<1> The Cumulative-to-Delta Processor requires a stateful collector deployment, therefore we recommend to deploy it as a sidecar to the application that emits the delta temporality metrics.
```

Comment:
@pavolloffay, would the following change make sense?
```
<1> The Cumulative-to-Delta Processor requires a stateful Collector deployment, which is best achieved by deploying it as a sidecar to the application that emits the delta temporality metrics.
```

Reply:
yes